src/Data/Voting.hs

```diff
@@ -50,12 +52,13 @@ instance Eq v => Category (DCat v) where
       | b0 == b1  = DelegationCat a c
       | otherwise = NoDelegation
 
+
 -- * pure model
 
 data DelegationMap = DelegationMap {
         _delegationMap
             :: Map
-                Voter -- who delegates
+                (Voter, Topic) -- who delegates in which topic
```

Comment:
s/Topic/Scope/?  scopes extend down-path: if you delegate in a topic, you delegate in all ideas in that topic (unless something overrules there).